src/index.html

```diff
@@ -10,23 +10,41 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+    >
+      <div class="searchbar__container">
+        <input
+          class="searchbar searchbar--big"
+          type="search"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+    </form>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line between the two `form` elements to improve readability.